docs/Developer-Guide.md

```diff
@@ -588,6 +559,16 @@ $ sudo awk '{if (/^\[proxy\.kata\]/) {got=1}; if (got == 1 && /^.*enable_debug/)
 $ sudo install -o root -g root -m 0640 /tmp/configuration.toml /etc/kata-containers/
 ```
 
+### Enable debug console for the agent
+
+Add `agent.debug_console` to the guest kernel command line to allow the agent process to start a debug console. 
+```
+$ conf="/etc/kata-containers/configuration.toml"
+$ sudo sed -i 's/^kernel_params.*/kernel_params="agent.debug_console=true"/g' "${conf}"
```

Comment:
If I'm reading the `sed` command right I think it would override any `kernel_params` that might have been there so far.  Is it intentional?  Similar `sed` command lines in Developer's Guide usually capture and preserve any existing `kernel_params`, something like

`sed -i 's/^kernel_params="\(.*\)"/kernel_params="\1 agent.debug_console=true"/g' "${conf}"`

(Disclaimer: I haven't debugged this and it might be oversimplified - not sure for instance if you have to handle possible whitespace around '='.)

If wiping out existing configuration and starting anew is the idea please ignore this. :-)